sds_data_manager/lambda_code/SDSCode/create_schema.py

```diff
@@ -2,18 +2,35 @@
 
 import json
 import logging
+import os
 import sys
 
+import boto3
 import requests
-from SDSCode.database.database import engine
 from SDSCode.database.models import Base
+from sqlalchemy import create_engine
 
 # Logger setup
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
 logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
 
 
+def get_db_uri():
```

Comment:
What was the error related to config.py and database.py? These are being used in multiple places (soon including queries.py) so it'd be a lot better if we could import these

Reply:
yes. It was causing some issue with import. But I just re-try and have this in `database.py` might work. I will push that change.

Reply:
I pushed that change. Looks like issue I was having at the beginning was resolved by db_session parameter now. So I moved `get_db_uri()` back into `database.py` and import in lambda(s).